app/models/settings/campaign.rb

```diff
@@ -0,0 +1,19 @@
+module Settings
+  class Campaign < RailsSettings::Base
+    self.table_name = :settings_campaigns
+
+    # The configuration is cached, change this if you want to force update
+    # the cache, or call Settings::Campaign.clear_cache
+    cache_prefix { "v1" }
```

Comment:
[nonblocking] Should we make this value editable outside of code? I don't have a great suggestion here, but the comment here seems to imply that it's something we might want to change and I'm not sure cache invalidation should require a deploy.

Reply:
This is how the `SiteConfig` model always worked. When I wrote the generator for new settings tables I just moved this over. Once I'm done with the splitting per se I'm open to more refactorings, this one included. Deal?